content/articles/building-a-network-with-dynamic-and-static-routing-using-cisco-packet-tracer/index.md

```diff
@@ -0,0 +1,93 @@
+
+We have finished working on connecting the circuits with the computers, now we need to give each computer & circuit its own unique ip adreass. Follow the steps of giving each pc a unique address.
+
+- Click on pc0 >> click on desktop >> click on ip configuration >> set ipv4 to 192.168.0.2 >> click on subnet mask for a default mask >> set default gataway to 192.168.0.1
+- Click on pc1 >> click on desktop >> click on ip configuration >> set ipv4 to 192.168.0.3 >> click on subnet mask for a default mask >> set default gataway to 192.168.0.1
+- Click on pc2 >> click on desktop >> click on ip configuration >> set ipv4 to 192.168.1.2 >> click on subnet mask for a default mask >> set default gataway to 192.168.1.1
+- Click on pc3 >> click on desktop >> click on ip configuration >> set ipv4 to 192.168.1.3 >> click on subnet mask for a default mask >> set default gataway to 192.168.1.1
+
+Now we need to set the ip for each router.
+- Click on router0 >> click on config >> click on GigabitEthernet0/0/0 >> set ipv4 to 192.168.0.1 >> click on subnet mask for a default mask >> turn the port status on
+- Click on router1 >> click on config >> click on GigabitEthernet0/0/0 >> set ipv4 to 192.168.1.1 >> click on subnet mask for a default mask >> turn the port status on
+- Click on router0 >> click on config >> click on GigabitEthernet0/0/1 >> set ipv4 to 10.0.0.1 >> click on subnet mask for a default mask >> turn the port status on
+- Click on router1 >> click on config >> click on GigabitEthernet0/0/1 >> set ipv4 to 10.0.0.2 >> click on subnet mask for a default mask >> turn the port status on
+
+Now for the final step, we need to set the RIP for each router in order to connect the computers of each side togsther.
+
+- Click on router0 >> click on config >> click on RIP >> add the default gataway of each router & the ip that the routers connect with each other with (192.168.0.0 - 192.168.1.0 - 10.0.0.0)
+- Click on router1 >> click on config >> click on RIP >> add the default gataway of each router & the ip that the routers connect with each other with (192.168.0.0 - 192.168.1.0 - 10.0.0.0)
+
+The RIP of each router should look like ![this](/engineering-education/building-a-network-with-dynamic-and-static-routing-using-cisco-packet-tracer/rip-ip.png).
+
+
+Green arrows will indicate that the network is fully connected and it should look like ![this](/engineering-education/building-a-network-with-dynamic-and-static-routing-using-cisco-packet-tracer/green-arrows.png).
+
+### Static routing
+
+Static routing is used when we manually configure a router, and we are not supposed to use static routing if there is a need for change or reconfiguration of the network.
+
+Static routing is very similar to dynamic routing in cisco packet tracer, but instead of using the RIP option, this time we will work on the static option. The steps of making a static network has the same steps of dynamic so we may repeat the steps untill we reach the RIP option.

```

Comment:
```suggestion
Static routing is like dynamic routing in cisco packet tracer, but instead of using the RIP option, this time we will work on the static option. The steps of making a static network are the same steps used in dynamic.

```